lang/cs/Org.Apache.REEF.Bridge/Org.Apache.REEF.Bridge.vcxproj

```diff
@@ -41,6 +41,7 @@ under the License.
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <Keyword>ManagedCProj</Keyword>
     <RootNamespace>Org.Apache.REEF.Bridge</RootNamespace>
+    <WindowsTargetPlatformVersion Condition="'$(VisualStudioVersion)' >= '15.0'">10.0.14393.0</WindowsTargetPlatformVersion>
```

Comment:
It turns out the executable will run on earlier versions of windows if this update  [KB2999226](https://support.microsoft.com/kb/2999226) is installed. From the update download page: "This update allows Windows desktop applications that depend on the Windows 10 Universal CRT release to run on earlier Windows operating systems."